### ISSUE TITLE
Refatoração dos métodos set

### ISSUE THREAD

**Issue by gabjohann** [SkiereszDiego/Java-Caldeira#7]
Foi dado como exemplo a refatoração do método set e com os seguinte dizer: " A gente poderia refatorar um dos métodos set para ficar assim".

`
public void setModelo(String modelo){
       this.modelo = modelo;
 }
`

A mesma refatoração pode ser aplicada para ambos os métodos ao mesmo tempo!?

`
public void setMarca(String  marca){
       this.marca = marca;
 }
`

**Comment by mbritzke** [comment]
Não entendi muito bem a pergunta, a tua ideia é criar um método só que receba 2 parametros e atualize os valores?
Se for, não é uma boa porque você trava o método para sempre receber duas informações, teria validar essa entradas e aumentaria a complexidade de set simples.
(Foi mal a demora em te responder)

**Comment by gabjohann** [comment]
Não, minha dúvida é se a mesma refatoração aplicada no método setModelo poderia ser aplicada no setMarca, ou seja, o código ficaria assim:

`
public void setModelo(String modelo){
        this.modelo = modelo; 
}

public void setMarca(String  marca){
        this.marca = marca;
}
`


Na hora que abri a issue, o que eu pensei é que da forma apresentada na refatoração, em um código maior onde se tem n getters & setters ficaria mais limpo, digamos assim.

**Comment by mbritzke** [comment]
Ah, certo!
Pode aplicar a mesma refatoração para o método setMarca que fica bom.
